meetings/2023-12-13.md

```diff
@@ -0,0 +1,127 @@
+* SC: FYI, there was some coverage of dweb/social approaches in https://gitlab.com/bluesky-community1/decentralized-ecosystem that you might find useful, if not already in your radar.
+* SC: Happy to help if you'd like to go over Solid Protocol or other specs: https://solidproject.org/TR/
+
+
+### WIP Implementation Feedback
+
+* SC: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome.
+* eP: More of a question: been working on establishing mutual relationship between two social agents. Have a working flow based on SAI. Currently relies on SAI sending through a message (sms, email). I want to optimize copy/paste flow so started using [webshare API](https://www.w3.org/TR/web-share/). Wanted to ask if people have experience with [web share target](https://w3c.github.io/web-share-target/) because it's not consistence across platforms. 
+* VB: May want to ask this in solid/app-development chat.
+
+
+### Solid CG Chair Election Procedure
+URL: https://github.com/solid/specification/discussions/582
+
+* SC: Voting closed. Requested W3C Team (Community Process) to give us the anonymised ballot data that's to be computed: https://lists.w3.org/Archives/Public/public-solid/2023Dec/0016.html
+* eP: W3C takes holiday break next week?
+* SC: Yes, I think 18th. Officially. 
+* eP: If they find it inconvenient to make a PR and send you data instead that'd be okay with you making the PR and they just approve or comment on that PR. I don't want to delay because they don't feel comfortable. 
+* SC: They're aware of the procedure. If there were issues they'd have mentioned or would let us know or just send me the data anyway. I don't expect any slowdown. They give us the data, we compute, we give to them, they approve and announce it. 
+
+
+### i18n and n11n of resource identifiers
+URL: https://github.com/solid/specification/pull/575
+
+* SC: All, please review.
+* SC: We're considering using IRIs instead of URIs, having mapping around.. gets a bit complicated. There's a number of specs involved. I reviewed, would be good to have more reviews. I'll follow up on my part. I think WT did a great job detailing what implementers need to do, but something like this needs deeper reviews, not just thumbs up to challenge each detail because things can be horribly wrong when ??? There's a parallel discussion on WebID CG about this topic. However we decide here will also influence how we use WebIDs. We need to have some implementation feedback on this. WT did a good job detailing what should happen, but not sure to what extent existing implementations will conform, and we don't have tests or a way to verify it. Hesitant to throw something like this into the spec and hoping things go smoothly. There's an alternative approach not to introduce this. If we use internationalized strings, we need to detail the conversion as for example WT ??? It comes down to how much interest people have. 
```

Comment:
```suggestion
* SC: We're considering using IRIs instead of URIs, having mapping around.. gets a bit complicated. There's a number of specs involved. I reviewed, would be good to have more reviews. I'll follow up on my part. I think WT did a great job detailing what implementers need to do, but something like this needs deeper reviews, not just thumbs up, to challenge each detail because things can go horribly wrong when ??? There's a parallel discussion on WebID CG about this topic. However we decide here will also influence how we use WebIDs. We need to have some implementation feedback on this. WT did a good job detailing what should happen, but not sure to what extent existing implementations will conform, and we don't have tests or a way to verify it. Hesitant to throw something like this into the spec, hoping things go smoothly. There's an alternative approach not to introduce this. If we use internationalized strings, we need to detail the conversion as for example WT ??? It comes down to how much interest people have. 
```

Reply:
This shows why long lines can be problematic. My changes (of which there are several, mostly tiny) are not properly highlighted inline.